infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterControllerV1.kt

```diff
@@ -158,6 +166,15 @@ class FrameConverterControllerV1 @Autowired constructor(private val frameConvert
         return requests as List<Request>
     }
 
+    private fun <T : FrameConverterRequestV1> assertRequestSize(requests: List<T>) {
+        if (requests.size > maxBatchRequests) {
+            throw ExtApiExceptionV1(
+                message = "Too many requests in batch: maxCount=$maxBatchRequests",
```

Comment:
Koska tämä on potentiaalisesti Geoviitteen ulkopuolelle lähtevä virhe, ilmoittaisin tuon maksimimääränkin mieluiten luonnollisen kielen lauseena (esim. tyyliin `Too many requests in batch: Maximum batch size is $maxBatchRequests`)

Reply:
Ei tuo suoranaisesti mene mihinkään käyttöliittymällä luultavasti kuitenkaan. Itse en käyttäisi luonnollista kieltä mihinkään error-viestiin, koska missään automaattisessakin integraatiossa tuo on helpompi käsitellä insinöörimuotoisena. Toki jos muut virheet on proosaistettuja, niin yhteneväisyyden nimissä voisi toki tämäkin olla :thinking: